doc/changelogs/CHANGELOG_V17.md

```diff
@@ -44,6 +45,108 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="17.8.1"></a>
+
+## 2022-04-04, Version 17.8.1 (Current), @BethGriggs prepared by @juanarbol
```

Comment:
Since this includes a semver-minor the version should be 17.9.0.

Reply:
:facepalm: